src/view.js

```diff
@@ -967,7 +968,8 @@ view.ModelFactoryService = class {
             new tflite.ModelFactory(),
             new tf.ModelFactory(),
             new sklearn.ModelFactory(),
-            new cntk.ModelFactory()
+            new cntk.ModelFactory(),
+            new openvinoIR.ModelFactory()
```

Comment:
`new openvino.ModelFactory()`

Reply:
renamed

is the discussion resolved now?